src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -12,4 +30,15 @@ def destroy
     end
     redirect_back(fallback_location: root_path)
   end
+
+  private
+
+  def permitted_params
+    params.require(:comment).permit(:body, :parent_id)
+  end
+
+  def find_commented
+    commentable = [Project, Package, BsRequest].find { |klass| klass.name == params[:commentable_type] }
```

Comment:
It; was the first option, but we could have a problem with code injection ( hakiri )